kubeshop/botkube
# add report deployment connection init (botVer, k8sVer)

## Description

Changes proposed in this pull request:

- add report connection init (send botVer & k8sVer)

## Testing


- run agent (connected with cloud deployment)
- check if it report new status connecting
- check if it reports botVer & k8sVer (deployment_status)

## Related issue(s)


#587
